test/command/test_fluentd.rb

```diff
@@ -1608,4 +1608,16 @@ def create_config_include_dir_configuration(config_path, config_dir, yaml_format
                          "[info]: loading additional configuration file path=\"#{conf_dir}/child.yml\"")
     end
   end
+
+  test "allow --disable-input-metrics option" do
+    conf = <<CONF
+<system>
+</system>
+CONF
+    conf_path = create_conf_file('input-metrics.conf', conf)
```

Comment:
```suggestion
    conf_path = create_conf_file('empty.conf', '')
```

Do we need this config for this test?

Reply:
Thanks.